lib/rule-tester/rule-tester.js

```diff
@@ -266,6 +267,7 @@ function wrapParser(parser) {
         };
     }
     return {
+        $parser: parser,
```

Comment:
Could we make this a symbol instead so it won’t show up in property iteration? `Symbol.for(“eslint.RuleTester.parser”)`?

Reply:
sounds good! I used `Object.defineProperty()` to do this - for not require`rule-tester.js` in `linter.js`.

Reply:
I really don't want to use a property with a prefix; symbols are a much better choice for this use case.

You don't have to require anything additional, just do this in both files:

```js
const parserSymbol = Symbol.for("eslint.RuleTester.parser");
```

The `Symbol.for()` method creates or retrieves a symbol with the given name, so as long as you use the same argument, you'll get the same symbol in both places. ([details](https://github.com/nzakas/understandinges6/blob/master/manuscript/06-Symbols.md#sharing-symbols))

Reply:
addressed in https://github.com/eslint/eslint/pull/14720/commits/849464ee81927c799bb895add4d53d496bd2da18.
thanks for the reviewing!